### PR TITLE
commit  missing method for redis

### PR DESCRIPTION
**Pull request by CrazyZard**
commit  missing method `decr ` `decrby` `lindex`

### CODE REVIEW

**Comment by kevwan** [comment]
Thanks for your PR!

Please add unit tests for redis commands. And put the functions in alphabetical order.

Thanks.

**Comment by CrazyZard** [comment]
i've already done a code-review 
thanks

File: core/stores/redis/redis.go [hunk 1]
```diff
@@ -765,6 +795,21 @@ func (s *Redis) Llen(key string) (val int, err error) {
 	return
 }
 
+// Lindex is the implementation of redis lindex command.
+func (s *Redis) Lindex(key string, index int64) (val string, err error) {
```
Comment (kevwan) [inline core/stores/redis/redis.go:799]: please add unit test for this func.

Comment (CrazyZard) [reply core/stores/redis/redis.go:799]: sorry  I was a little careless
i've my check unit test and sort func

**Review by kevwan — APPROVED** [review]
lgtm

**Comment by CrazyZard** [comment]
goods news

**Pull request merged**